e2e/specs/patientRegistration.spec.ts

```diff
@@ -0,0 +1,27 @@
+import { test } from '../core';
+import { PatientRegistrationFormValues, PatientRegistrationPage } from '../pages/patientRegistrationPage';
+
+test('should be able to register a patient', async ({ loginAsAdmin: page, api }) => {
+  const patientRegistrationPage = new PatientRegistrationPage(page);
+
+  await patientRegistrationPage.goto();
+
+  const formValues: PatientRegistrationFormValues = {
+    givenName: 'Johnny',
+    middleName: 'Donny',
+    familyName: 'Ronny',
+    sex: 'male',
+    birthdate: '2020-2-1',
+    postalCode: '',
+    address1: 'Bom Jesus Street',
+    address2: '',
+    country: 'Brazil',
+    countyDistrict: 'Antônio dos Santos',
+    stateProvince: 'Pernambuco',
+    cityVillage: 'Recife',
+    phone: '5555551234',
+    email: 'jodon@example.user.com',
+  };
+
+  await patientRegistrationPage.fillPatientRegistrationForm(formValues);
```

Comment:
Upon completion of the form, it is essential to validate that the patient has been registered correctly. This can be accomplished by verifying the following:

- The system should redirect the user to the patient chart page.
- The displayed information (such as name, address, birthday, and contact details) should be to match the inputted data."